src/test/scala/org/ergoplatform/reemission/ReemissionRulesSpec.scala

```diff
@@ -210,4 +212,38 @@ class ReemissionRulesSpec extends ErgoPropertyTest with ErgoTestConstants {
     prover.prove(emptyEnv, pay2RewardsProp, ctx, fakeMessage).isFailure shouldBe true
   }
 
+  def compareSizes(tree: ErgoTree, name: String) = {
+    println(s"\n\n====  original $name  ================================")
+    println(s"size of $name: ${tree.bytes.length}")
+//    SigmaPPrint.pprintln(tree, 150, 250)
+
+    val IR.Pair(calcF, _) = IR.doCosting(Interpreter.emptyEnv, tree.toProposition(true), true)
+    val compiledProp = IR.buildTree(calcF).toSigmaProp
+    val version: Byte = 1
+    val headerFlags = ErgoTree.headerWithVersion(version)
+    val compiledTree = ErgoTree.fromProposition(headerFlags, compiledProp)
+
+    println(s"====  compiled $name  ================================")
+    println(s"size of $name: ${compiledTree.bytes.length}")
+//    SigmaPPrint.pprintln(compiledTree, 150, 250)
+  }
+
+  property("compile reemission scripts") {
```

Comment:
This is not a real test, rather, an utility printing something. so should be done as a *Printer utility in org.ergoplatform.tools. Also, I do not understand what is it doing. Comments are needed.

Reply:
yes, was just to understand the effect of compilation.
I think this PR can just be closed without merging.